doc/backend_vars.asciidoc

```diff
@@ -16,7 +16,10 @@ _EXIT_AFTER_SCHEDULE;boolean;0;Exit test execution immediately after evaluation
 _SKIP_POST_FAIL_HOOKS;boolean;0;Skip the execution of post_fail_hook methods if set. This can be useful to save test execution time during test development when the post_fail_hook is not expected to provide any value as most likely the test developer already knows what needs to be done as a next step on a test fail. Note that this also skips the test module result evaluation, for example a failing fatal test module does not abort the isotovideo run.
 TEST_GIT_REFSPEC;string;;git refspec to checkout within `CASEDIR` when `CASEDIR` is a git working copy. By default, does not change the content of `CASEDIR`. Overrides the optional git refspec in `CASEDIR`. Can be used to explicitly select a git commit within an existing git working copy and also to skip unnecessary git network transfers when `CASEDIR` is already providing the right git working copy.
 NEEDLES_GIT_REFSPEC;string;;git refspec to checkout within `NEEDLES_DIR`. See `TEST_GIT_REFSPEC` for details.
-NOVIDEO;boolean;0;Do not encode video if set
+EXTERNAL_VIDEO_ENCODER_CMD;string;;Specifies the command line for invoking a custom video encoder. It which is supposed to accept a sequence of PPM images via stdin. The placeholder `%OUTPUT_FILE_NAME%` is replaced with the output file path. The output file path is appended if it is missing. Example: `ffmpeg -r 24 -f image2pipe -vcodec ppm -i - -c:v libvpx-vp9 -crf 35 -b:v 1500k -cpu-used 0`
+EXTERNAL_VIDEO_ENCODER_OUTPUT_FILE_EXTENSION;string;webm;The extension of the output file when `EXTERNAL_VIDEO_ENCODER_CMD` is used.
+EXTERNAL_VIDEO_ENCODER_ADDITIONALLY;boolean;0;Whether `EXTERNAL_VIDEO_ENCODER_CMD` should only be invoked additionally to the built-in Theora encoder. (Configuring an external video encoder disables the built-in one by default.)
+NOVIDEO;boolean;0;Do not encode video if set; `EXTERNAL_VIDEO_ENCODER_` variables are ignored as well.
```

Comment:
I wonder, how is this additional ";" rendered. Will it add another column to the complete table?

Reply:
No, it apparently screws up the table completely as the cell will be part of the next row. Good catch.